src/RolloutScorer/RolloutScorer/RolloutScorer.cs

```diff
@@ -158,13 +167,13 @@ public async Task<(int numHotfixes, int numRollbacks)> CalculateNumHotfixesAndRo
                 BuildSource source = (await GetAzdoApiResponseAsync(build.BuildSummary.SourceLink)).ToObject<BuildSource>();
 
                 // we can only automatically calculate rollbacks if they're tagged; so we specifically don't try when --assume-no-tags is passed
-                if (!AssumeNoTags && source.Comment.StartsWith(AzureDevOpsCommitTags.RollbackTag, StringComparison.InvariantCultureIgnoreCase))
+                if (!AssumeNoTags && source.Comment.Contains(AzureDevOpsCommitTags.RollbackTag, StringComparison.InvariantCultureIgnoreCase))
                 {
                     numRollbacks++;
                     build.Score.Rollbacks = 1;
                 }
                 // if we're assuming no tags, every deployment after the first is assumed to be a hotfix; otherwise we need to look specifically for the hotfix tag
-                else if (AssumeNoTags || source.Comment.StartsWith(AzureDevOpsCommitTags.HotfixTag, StringComparison.InvariantCultureIgnoreCase))
+                else if (AssumeNoTags || source.Comment.Contains(AzureDevOpsCommitTags.HotfixTag, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
>n [](start = 58, length = 1)

Contains is a LOT more dangerous than starts with. It means if I accidentally mention this take anywhere, it'll get tagged...  I though we were going to use labels for these things instead of scraping comments so we didn't have these kinds of bugs lurking around?

Reply:
This is for commit messages, so labels don't work. Contains is necessary for those because the commit message will never start with the square-bracket tag. I think one thing I'll do is at least force it to be case-sensitive which should reduce the kinds of error's you're talking about. These ones still have to be things like "[HOTFIX]" and "[ROLLBACK]", etc., so the square brackets should also protect us for the most part.

Reply:
To clarify, they won't start with it because instead it will be something like "Merged PR blah" for the most part.

Reply:
I _always_ remote the "Merged PR" text from my commit messages because it's useless nonsense. :-) I wish more people would, since an endless list of "Merged PR some-arbitrary-number" makes it _really_ hard to scan commit messages. ;-)

Reply:
The commit messages are only `Merged PR <stuff>` in azure devops. Github just adds the pr number at the end of the message so it doesn't get in the way.